mautic/mautic
# Multiple select custom fields can be created without errors

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [&check;]
| New feature/enhancement? (use the a.x branch)      | [&cross;]
| Deprecations?                          | [&cross;]
| BC breaks? (use the c.x branch)        | [&cross;]
| Automated tests included?              | [&check;] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12208  



#### Description:
When a custom field (type select or select - multiple) is created, a 500 error is thrown. This PR fixes the error.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a custom field, select the type of select or select - multiple.
3. Add options and save.
4. Edit the field.
5. Works now. The 500er was before.
